### DISCUSSION TITLE
How to insert the vCard value?

### DISCUSSION THREAD

**Discussion by NumanTalukder** [zpao/qrcode.react#286] [Q&A] [answered]
Hi, I'm trying to insert the value of a vCard into my project, and I'm not sure how to do it. Can anyone provide guidance or point me in the right direction? Thanks!

**Comment by zpao** [comment, marked as answer]
This works out of the box, just passing the vcard content in as `value` (with newlines). I just tested with the example from Wikipedia on https://zpao.github.io/qrcode.react/.
